flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/catalog/CatalogCalciteSchema.java

```diff
@@ -42,10 +43,16 @@
 	private final CatalogManager catalogManager;
 	// Flag that tells if the current planner should work in a batch or streaming mode.
 	private final boolean isStreamingMode;
+	private final SqlExprToRexConverterFactory converterFactory;
```

Comment:
Could you add a short comment for this field to explain why we need this? The same to `CatalogManagerCalciteSchema`, `CatalogSchemaTable`, `DatabaseCalciteSchema`. For example:

```
The SQL expression converter factory is used to derive correct result type of computed column, because the date type of computed column from catalog table is not trusted. 
```